packages/pageflow/src/frontend/page_types/mixins/common_page_css_classes.js

```diff
@@ -5,6 +5,7 @@ pageflow.commonPageCssClasses = {
   updateCommonPageCssClasses: function(pageElement, configuration) {
     pageElement.toggleClass('invert', configuration.get('invert'));
     pageElement.toggleClass('hide_title', configuration.get('hide_title'));
+    pageElement.toggleClass('hide_logo', configuration.get('hide_logo'));
```

Comment:
`toggleClass` has the unpleasant behavior to toggle the class on every call if the second parameter is `undefined` instead of `false`. The two lines above, therefore depend on the fact that `invert` and `hide_title` both have [boolean defaults](https://github.com/codevise/pageflow/blob/9872a63d8e7b527b4cfaaa6eaeb8019fc090b3d3/packages/pageflow/src/editor/models/Configuration.js#L22). The safer thing would have been to pass `!!configuration.get('hide_title')`. The double bang turns `undefined` into `false`. This is done is most other places using `toggleClass` in the code base.